queue_services/entity-emailer/src/entity_emailer/email_processors/consent_continuation_out_notification.py

```diff
@@ -0,0 +1,159 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Email processing rules and actions for CCO notifications."""
+from __future__ import annotations
+
+import base64
+import re
+from http import HTTPStatus
+from pathlib import Path
+
+import requests
+from entity_queue_common.service_utils import logger
+from flask import current_app
+from jinja2 import Template
+from legal_api.models import Business, Filing, UserRoles
+
+from entity_emailer.email_processors import (
+    get_filing_info,
+    get_recipient_from_auth,
+    substitute_template_parts,
+)
+
+
+def _get_pdfs(
+        status: str,
+        token: str,
+        business: dict,
+        filing: Filing,
+        filing_date_time: str,
+        effective_date: str) -> list:
+    # pylint: disable=too-many-locals, too-many-branches, too-many-statements, too-many-arguments
+    """Get the pdfs for the incorporation output."""
```

Comment:
Consent Continuation Out

Reply:
Good catch! I copied this from the dissolution class, so it must've been wrong there. I checked to make sure there were no references to dissolutions in the CCO class.